### PR TITLE
Add acceptance tests

### PR DESCRIPTION
**Pull request by mcanevet**
Somehow the puppet service does not start on Debian and I'm looking for a way to highlight.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,6 +12,19 @@ env:
   - PUPPET_VERSION=3.5 FUTURE_PARSER=yes TRUSTED_NODE_DATA=yes
   - PUPPET_VERSION=4.0
 matrix:
+  include:
+    - rvm: default
+      sudo: required
+      services: docker
```
Comment (ekohl) [inline .travis.yml]: Can you use docker on travis inside a container? We use sudo: false and IIRC that was not (yet) supported.

File: .travis.yml [hunk 2]
```diff
@@ -12,6 +12,19 @@ env:
...
+      env: BEAKER_set="debian-8-x86_64-docker"
+      bundler_args:
+      script: bundle exec rspec spec/acceptance/*_spec.rb
+    - rvm: default
+      sudo: required
+      services: docker
```
Comment (mcanevet) [inline .travis.yml]: @ekohl yes with this 2 lines

**Comment by mcanevet** [comment]
@ekohl FYI : http://www.camptocamp.com/en/actualite/puppet-acceptance-tests-travis-ci-docker/

**Comment by ekohl** [comment]
Cool stuff

**Comment by mcanevet** [comment]
Indeed

**Comment by mmoll** [comment]
please rebase :)

**Comment by mmoll** [comment]
I guess this would need `sudo: true` for travis?

**Comment by brandonweeks** [comment]
@mmoll I'm only seeing timeouts?

**Comment by mmoll** [comment]
@brandonweeks unfortunately some of our tests are known to be timing out (10 mins no output or 2h overall) at the moment. The new stuff here are the two "BEAKER_set..." ones at the bottom of the list.

**Comment by mmoll** [comment]
What's the status here? The two added test runs failed, also this needs a rebase :)

**Comment by mmoll** [comment]
no reaction, closing

**Pull request closed without merging**